### PR TITLE
IaC scanning

### PR DESCRIPTION
**Pull request by drichards-87**
### What does this PR do? What is the motivation?

Short article for IaC scanning private beta. Requested by PM.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/security/cloud_security_management/iac_scanning.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: IaC Scanning
+private: true
+---
+
+<div class="alert alert-info">Static Infrastructure as Code (IaC) scanning is in private beta.</div>
```
Comment (brett0000FF) [inline content/en/security/cloud_security_management/iac_scanning.md:6]:
```suggestion
<div class="alert alert-info">Static Infrastructure as Code (IaC) scanning is in Preview.</div>
```
Just noting we can say Preview as long as it doesn't say Private Beta in the UI.

File: content/en/security/cloud_security_management/iac_scanning.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+### Enable IaC scanning for your repositories
+
+After you set up the GitHub integration, enable IaC scanning for the repositories in your GitHub account.
+
+1. On the [CSM Setup page][4], expand the **Source Code Integrations** section.
+2. Click **Configure** for the GitHub account you want to configure.
+3. To enable IaC scanning:
+    - All repositories: Toggle **Enable Infrastructure as Code (IaC) Scanning** to the on position.
+    - Single repository: Toggle the **IAC Scanning** option for the specific repository to the on position.
```
Comment (brett0000FF) [inline content/en/security/cloud_security_management/iac_scanning.md:34]:
```suggestion
    - Single repository: Toggle the **IAC** option for the specific repository to the on position.
```
Just based on a quick glance in Demo, it appears that this column is just "IAC".

**Review by brett0000FF — APPROVED** [review]
Looks good! I just had a few minor thoughts (added as suggestions).

File: content/en/security/cloud_security_management/iac_scanning.md (hunk 2 above)
Comment (drichards-87) [reply content/en/security/cloud_security_management/iac_scanning.md:34]: IAC is for remediation. For scanning, the toggle is labeled "IAC Scanning". But that's a good point, though. It's a little confusing to have IAC and IAC Scanning toggles. I'll pass that along to the PMs.

**Pull request merged**